installation/gcp/dm/vmclarity-config.yaml

```diff
@@ -0,0 +1,16 @@
+imports:
```

Comment:
Do we need this file? Why are the values of  sshPublicKey and the images are hard coded?

Reply:
I've removed this one, and left a vmclarity-config.example.yaml instead, and updated the README to describe how to use it.